app/src/main/java/unipd/se18/ocrcamera/inci/Inci.java

```diff
@@ -0,0 +1,81 @@
+package unipd.se18.ocrcamera.inci;
+
+import android.util.Log;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+
+import com.opencsv.CSVReader;
+
+/**
+ * Class for Inci database management
+ * The database is in CSV format, using openCSV to parse the file.
+ * For more informations about openCSV:
+ * http://opencsv.sourceforge.net/
+ * @author Francesco Pham
+ */
+public class Inci {
+
+    private static final String TAG = "Inci";
```

Comment:
For a better look i advice to add all string in string.xml file in resources and then retrive it when u need. Also better if u want to translate the app in the future